CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -83,6 +83,10 @@ EnableTutorialDesc="Enable tutorial"
 EnableTutorialTooltip="Show in-game explanations and tips regarding changes and new features/mechanics"
 RemoveNicknamedUpgradesDesc="Remove upgrades from nicknamed items"
 RemoveNicknamedUpgradesTooltip="Disabling this will preserve any nicknamed weapon's upgrades when selecting 'MAKE UPGRADES AVAILABLE' at the squad select screen"
+WarnBeforeExpirationDesc="Warn Before Expiration"
+WarnBeforeExpirationTooltip="Warn before an activity is about to expire"
+HoursBeforeWarningDesc="Hours Before Warning"
+HoursBeforeWarningTooltip="How many hours left before warning"

```

Comment:
Not thrilled about wording, but can't really say what's wrong with it....

cc @cannonfodder1 wdyt?

Reply:
Titles are fine the tooltips need some tiny changes.
"Show a warning before an activity is about to expire"
"The warning will appear this many hours before expiry"